cmake/external/onnxruntime_external_deps.cmake

```diff
@@ -542,8 +542,11 @@ if(TARGET ONNX::onnx_proto AND NOT TARGET onnx_proto)
   message(STATUS "Aliasing ONNX::onnx_proto to onnx_proto")
   add_library(onnx_proto ALIAS ONNX::onnx_proto)
 endif()
-
-include(external/eigen.cmake)
+if(onnxruntime_USE_VCPKG)
```

Comment:
VCPKG is only used on Windows, so this change broke Linux builds using system Eigen.
The right variable here is "onnxruntime_USE_PREINSTALLED_EIGEN".

Reply:
VCPKG can support all platforms.

Reply:
The onnxruntime_USE_PREINSTALLED_EIGEN variable is deleted.

Reply:
I think we may change onnxruntime_USE_VCPKG to "onnxruntime_PREFER_FIND_PACKAGE"  if you don't want to use VCPKG you still can use this var?

Reply:
> The onnxruntime_USE_PREINSTALLED_EIGEN variable is deleted.

No, it's still here: https://github.com/microsoft/onnxruntime/blob/989d4177ed99db324ba4a4a35149977626120b14/cmake/CMakeLists.txt#L93